components/http_client.rst

```diff
@@ -477,6 +478,14 @@ errors, exceptions will notify you when needed. On the other hand, if you write
 the error-handling code, you will opt-out from these fallback mechanisms as the
 destructor won't have anything remaining to do.
 
+There are two types of exceptions:
```

Comment:
there is also `DecodingExceptionInterface` :)

Reply:
Added.